src/routes/documents/[id]-[slug]/sidebar/Sections.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import type { Section } from "$lib/api/types";
+
+  import { ListOrdered16 } from "svelte-octicons";
+
+  import Empty from "@/lib/components/common/Empty.svelte";
+  import SidebarGroup from "@/lib/components/sidebar/SidebarGroup.svelte";
+  import SidebarItem from "@/lib/components/sidebar/SidebarItem.svelte";
+
+  export let sections: Section[];
+</script>
+
+<SidebarGroup>
+  <h2 slot="title">
+    <ListOrdered16 />
+    Sections
+  </h2>
+
+  {#each sections as section}
+    <SidebarItem>
+      {section.title}
+    </SidebarItem>
+  {:else}
+    <Empty icon={ListOrdered16}>
```

Comment:
```suggestion
    <Empty icon={ListOrdered24}>
```

I've been using the 24px variants for empty sections, since their line weight is slightly lighter.